front-end/src/components/Admin/Grid/index.js

```diff
@@ -100,16 +102,33 @@ class Grid extends Component {
     const { entityType, tabKey } = this.props.match.params;
     const { setTabContentUrl } = this.props;
     const { currentPage, pagesTotal } = this.state.meta;
+    const paginationPages = [];
+
     setTabContentUrl(entityType);
 
+    for (let number = 1; number <= pagesTotal; number++) {
+      paginationPages.push(
+        <Pagination.Item active={number === currentPage}
+          onClick={() => this.getData(number, elementsPerPage)}>{number}</Pagination.Item>
```

Comment:
From eslint:
/home/travis/build/Thealoner/danit-final-project/front-end/src/components/Admin/Grid/index.js
  112:20  warning  Don't make functions within a loop  no-loop-func
  112:47  error    'elementsPerPage' is not defined    no-undef